docs/reference/commandline/volume_create.md

```diff
@@ -0,0 +1,42 @@
+<!--[metadata]>
++++
+title = "volume create"
+description = "The volume create command description and usage"
+keywords = ["volume, create"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# volume create
+
+    Usage: docker volume create [OPTIONS]
+
+    Create a volume
+
+    -d, --driver=local    Specify volume driver name
+    --help=false          Print usage
+    --name=               Specify volume name
+    -o, --opt=map[]       Set driver specific options
+
+Creates a new volume that containers can can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
+
+  $ docker volume create --name hello
+  hello
+  $ docker run -d -v hello:/world busybox ls /world
+
+The mount is created inside the container's `/src` directory. Docker doesn't not support relative paths for mount points inside the container. 
+
+Multiple containers can use the same volume in the same time period. This is useful if two containers need access to shared data. For example, if one container writes and the other reads the data.
+
+## Driver specific options
+
+Some volume drivers may take options to customize the volume creation. Use the `-o` or `--opt` flags to pass driver options:
+
+  $ docker volume create --opt size=10G
```

Comment:
I'll add this because youll be updating anyway; could you add a "fake" driver here, and possibly a more fake looking option to prevent people *trying* this; they might miss that it's not supported. (Also see my comment on this line from yesterday) <3